smartprom.py

```diff
@@ -164,9 +164,9 @@ def collect():
     for drive, drive_attrs in DRIVES.items():
         typ = drive_attrs['type']
         try:
-            if typ == 'sat':
+            if typ == 'sat' or typ == 'usbjmicron' or typ == 'usbprolific' or typ == 'usbsunplus':
```

Comment:
Could you change the code to something like this?
```
sat_types = ['sat', 'usbjmicron', 'usbprolific', 'usbsunplus']
nvme_types = ...
if typ in sat_types:
 ...
```

Reply:
Yes, of course.

Reply:
I will do it in another PR.